dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/test-applications/EL50BasicTests.war/src/io/openliberty/el50/fat/basic/servlets/EL50BasicTestsServlet.java

```diff
@@ -95,19 +100,36 @@ public void testGenerics() throws Exception {
      * @throws Exception
      */
     @Test
-    @Mode(TestMode.FULL)
     public void testGetFeatureDescriptors_returnsNull() throws Exception {
         ELResolver resolver = new CustomELResolver();
         assertNull("The result was expected to be null for ELResolver getFeatureDescriptors but was not.",resolver.getFeatureDescriptors(new StandardELContext(ELManager.getExpressionFactory()),new Object()));
     }
 
-
-    public class SimpleBean {
-
-        public int getNumber() {
-            return 25;
-        }
-
+    /**
+     *
+     * Expression Language 5.0 in Jakarta EE10 clarified and documented getType must return null if property is read-only
+     * and switched implementation of StaticFieldELResolver getType to match this behavior as it was inconsistent before EL 5.0.
+     * https://github.com/jakartaee/expression-language/issues/168
+     * @throws Exception
+     */
+    @Test
+    public void testGetType_returnsNull() throws Exception {
+        ELManager elm = new ELManager();
+        StaticFieldELResolver resolver = new StaticFieldELResolver();
+        elm.addELResolver(resolver);
+        ELContext context = elm.getELContext();
+        context.setPropertyResolved(false);
+        Object bean = new ELClass(SimpleBean.class);
+        Class<?> type = resolver.getType(context, bean, "staticString");
+        assertTrue("When calling getType for StaticFieldELResolver, the property was not resolved!", context.isPropertyResolved());
+        assertNull("When calling getType for StaticFieldELResolver, did not return null instead returned: " +type, type);
```

Comment:
"When calling getType for StaticFieldELResolver, the returned value was not null and instead was:" also add a space after `+`

Reply:
I'll work on it